sdk/python/kfp/compiler/_container_builder.py

```diff
@@ -18,19 +18,37 @@
 import os
 import uuid
 
+SERVICEACCOUNT_NAMESPACE = '/var/run/secrets/kubernetes.io/serviceaccount/namespace'
+GCS_STAGING_BUCKET_DEFAULT = 'gs://kfp_container_build_staging'
+
 class ContainerBuilder(object):
   """
   ContainerBuilder helps build a container image
   """
-  def __init__(self, gcs_staging, namespace):
+  def __init__(self, gcs_staging=GCS_STAGING_BUCKET_DEFAULT, namespace=None):
     """
     Args:
-      gcs_staging (str): GCS blob that can store temporary build files
+      gcs_staging (str): GCS bucket/blob that can store temporary build files,
+          default is GCS_STAGING_BUCKET_DEFAULT defined above.
```

Comment:
nit: better to put the real value instead of the constant name here, as user probably won't be able to find the constant value in notebook.

Reply:
SG